modules/engines/engine-core/src/main/java/com/sap/xsk/modificators/CalculationViewDataSourceTransformation.java

```diff
@@ -30,6 +30,14 @@ public class CalculationViewDataSourceTransformation {
       + "            <xsl:apply-templates select=\"@*|node()\"/>\n"
       + "        </xsl:copy>\n"
       + "    </xsl:template>\n"
+      + "    <xsl:template match=\"logicalJoin/@associatedObjectUri\">\n"
+      + "         <xsl:attribute name=\"associatedObjectUri\">\n"
+      + "           <xsl:value-of select=\"concat(substring-before(substring-after(., '/'), '/'), '::', substring-after(substring-after(substring-after(., '/'), '/'), '/'))\" disable-output-escaping=\"yes\"  />\n"
+      + "         </xsl:attribute>\n"
+      + "     </xsl:template>\n"
+      + "     <xsl:template match=\"measureMapping/@schemaName\">\n"
+      + "         <xsl:copy-of select=\"/@*[name(.)!='schemaName']|node()\" />\n"
+      + "     </xsl:template>\n"
       + "</xsl:stylesheet>";
 
```

Comment:
i think would be better to rename CalculationViewDataSourceTransformation -> CalculationViewTransformation as from now we would make changes not only on data source artefact.

Reply:
Ok. Committed.